x-pack/plugins/ml/public/application/model_management/test_models/models/question_answering/question_answering_inference.ts

```diff
@@ -46,6 +46,7 @@ export type QuestionAnsweringResponse = InferResponse<
 >;
 
 export class QuestionAnsweringInference extends InferenceBase<QuestionAnsweringResponse> {
+  static initialQuestionText?: string;
```

Comment:
We should find a different way to solve this problem and not use static variables. Static variables shouldn't be used to persist information like this. They should be used to allow a class to share constant information related to the class or static util functions which require no instance of the class to exist.

Using static vars for this information causes the value to exist for the lifetime of the page. If I enter a question to test a question answering model, when I come to test a different question answering model, my previously entered text is remembered.